pkg/appAPIs/appAPIs.go

```diff
@@ -407,19 +444,46 @@ func Login(token string) error {
 }
 
 // Generate environemnt slice as per create command. [{ "key":"ENV1", "value":"val1"}, { "key":"ENV2", "value":"val2"}]
-func genEnvSlice(env []string) []string {
+func genEnvSlice(env []string) ([]string, map[string]string) {
 	var envSlice []string
-
+	sliceMap := make(map[string]string)
 	if env != nil {
 		for _, value := range env {
 			splitEnv := strings.Split(value, "=")
+			sliceMap[splitEnv[0]] = splitEnv[1]
 			envSlice = append(envSlice, fmt.Sprintf(`{"key": "%v", "value": "%v"}`, splitEnv[0], splitEnv[1]))
 		}
 	}
 	for count := 0; count < len(envSlice)-1; count++ {
 		envSlice[count] = envSlice[count] + ","
 	}
-	return envSlice
+	return envSlice, sliceMap
+}
+
+// Generate environemnt slice from env File. [{ "key":"ENV1", "value":"val1"}, { "key":"ENV2", "value":"val2"}]
+func GetSliceFromEnvFile(envFilePath string) ([]string, map[string]string, error) {
+	var envSlice []string
+	envMap := make(map[string]string)
+	if envFilePath != "" {
+		envFile, err := os.Open(envFilePath)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error opening the env file. Please make sure that file path: %s is valid.", envFilePath)
+		}
+		defer envFile.Close()
+		scanner := bufio.NewScanner(envFile)
+		for scanner.Scan() {
+			text := scanner.Text()
+			splitEnv := strings.Split(text, "=")
```

Comment:
What if data in `.env` file is not of specified format? i.e not of Key=Value. Shouldn't we error out with a proper error message.

Reply:
handled this one
<img width="899" alt="wrong_format" src="https://user-images.githubusercontent.com/49315586/160340073-141a3a6f-43ad-4888-bc5f-0febdce18b7e.png">